### DISCUSSION TITLE
I really like the new material you palette generator.

### DISCUSSION THREAD

**Discussion by J43git** [T8RIN/ImageToolbox#806] [General]
It's amazing how within the span of the last few months this app can replace so many others I have. I still haven't upgraded to Android 12 yet but one of the things I always wanted was to use the palette generator for the dynamic theme. This recent beta gives that to me. Not only that but I can make the palette themed after all the filters that mainline android still hasn't implemented fully, so it's pretty neat. Once again, I really praise you for always adding something to updates, whether in alpha or beta. I thank you, and I hope the app continues to soar!

**Comment by T8RIN** [comment]
Thanks, doing my best :smile:
